app/templates/courses/teacher-courses-view.jade

```diff
@@ -93,3 +94,7 @@ mixin course-info(course)
           | (
           span(data-i18n='teacher.guides_coming_soon')
           | )
+    if campaign && campaign.get('levelsUpdated')
+      p.small.m-t-2
+        span.spr(data-i18n="courses.last_updated")
```

Comment:
Trailing whitespace is worse than the copy-paste issue, I think. It's far too easy to remove significant trailing whitespace without realizing it. I'd rather continue using .spr and .spl and migrating from there to a better solution, than have a mix.